docs/SBBR_recipe_testlist.md

```diff
@@ -1927,3 +1927,6 @@ Refer to [UEFI SCT Test Case Specification](https://uefi-sct-testcasespec.readth
 </tr>
 </tr>
 </table>
+
+**Note:**
```

Comment:
Can we move the note to the top? 
Can we clarify the meaning of "Arm specific tests"? Some of the marked tests can be run on the systems based on other architecutres. If it means the tests are only in the bbr-acs repo and have not yet been upstreamed to edk2-test, can we update the wording to clarify this?